lib/datadog/ci/ext/environment.rb

```diff
@@ -135,12 +139,15 @@ def extract_bitbucket(env)
             Datadog::Ext::Git::TAG_REPOSITORY_URL => env['BITBUCKET_GIT_SSH_ORIGIN'],
             Datadog::Ext::Git::TAG_TAG => env['BITBUCKET_TAG'],
             TAG_JOB_URL => url,
-            TAG_PIPELINE_ID => env['BITBUCKET_PIPELINE_UUID'] ? env['BITBUCKET_PIPELINE_UUID'].tr('{}', '') : None,
+            TAG_PIPELINE_ID => env['BITBUCKET_PIPELINE_UUID'] ? env['BITBUCKET_PIPELINE_UUID'].tr('{}', '') : nil,
```

Comment:
Was this a bug, or was this constant defined somewhere?

Reply:
Copy/paste from Python, a bug for sure.